src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/data-governance.mdx

```diff
@@ -7,7 +7,151 @@ tags:
 metaDescription: How to manage your data from the Kubernetes integration.
 freshnessValidatedDate: never
 ---
+## Logs
+The collection of  [Logs](https://github.com/newrelic/helm-charts/blob/master/charts/newrelic-logging/values.yaml#L27) in Kubernetes is  done  using the [New Relic kubernetes plugin](https://docs.newrelic.com/docs/logs/forward-logs/kubernetes-plugin-log-forwarding/), an output plugin for Fluentbit.
 
+The configuration of **fluentbit** includes:
+- **Input Plugin**: This is where we define the log sources that we wish to collect, as in our case we are talking about k8s logs.
+- **Parser Plugins**: to convert and structure log messages (JSON, Regexp, LTSV, Logfmt, etc.)
+- **Filter Plugins**: to modify, enrich and delete   information from your logs (Nest, Throttle, Expect, Geoplp, Grep, K8s, etc.) For filtering, you can also use a LUA script.
+- **Output Plugins**: send the log stream to several outputs which will be responsible for storing the log stream and displaying them. In this case, we speak of NRDB (New Relic Database).
+
+Logs can be filtered following the methods below:
+### Change the path
```

Comment:
@wadhah-nr The "Change path" / "Exclude paths" / "Pod annotations" 3 separate procedures customers choose from, or is it a 3 step procedure that customers need to work through sequentially?

Reply:
@akristen yes exactly: those represent 3 different ways/options for customers to optimise log data ingestion.

Reply:
@wadhah-nr When do we recommend a certain method to certain customers? We like to give some guidance on when a customer's use case might fit one method over another. In other words, when might a customer choose:

* Changing the path over the other two?
* Excluding the path over the other two?
* Using pod annotations?

Reply:
A follow up question: Does New Relic prefer one method over all other use cases? if so, why?